File: src/ui/app/root/application/models/Admission/EligibilityCriteria.ts
export interface IAdmissionEligibilityCriteria {
	eligibilityCriteriaId : string;
	campusProgramId : string;
	admissionTypeId : string;
	genderId : string;
	markPercentage : number;
	minPassingYear : Date;
	fromDob : Date;
	toDob : Date;
	statusId : number;
	loggerId : string;

}

export interface IAdmissionEligibilityCriteriaVM {
    eligibilityCriteriaId: string;
    campusProgramId: string;
    admissionTypeId: string;
    genderId: string;
    markPercentage: number;
    minPassingYear: string;
    fromDob: string;
    toDob: string;
    statusId: number;
    fullName: String;
    gender: String;
    description: String;
    sessionId:string;
    campusId:string;
    loggerId: string;
}

File: src/data/Model/Cms360/Admission/EligibilityCriteria.cs
/*
*   Auther: H.Muhammad Kamran
*   email: hmuhdkamran@gmail.com
*   contact: +92 (313 / 333) 9112 845
*/

using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Cms360.Data.Model
{
    [Table("EligibilityCriteria", Schema = "Admission")]
    public partial class AdmissionEligibilityCriteria
    {
		[Key]
		[Required]
		public Guid EligibilityCriteriaId { get; set; }
		[Required]
		public Guid CampusProgramId { get; set; }
		[Required]
		public Guid AdmissionTypeId { get; set; }
		[Required]
		public Guid GenderId { get; set; }
		[Required]
		public int MarkPercentage { get; set; }
		[Required]
		public DateTime MinPassingYear { get; set; }
		[Required]
		public DateTime FromDob { get; set; }
		[Required]
		public DateTime ToDob { get; set; }
		[Required]
		public int StatusId { get; set; }
		[Required]
		public Guid LoggerId { get; set; }

    }

	 [Table("VW-EligibilityCriteria", Schema = "Admission")]
    public partial class AdmissionEligibilityCriteriaVM
    {
		[Key]
		[Required]
		public Guid EligibilityCriteriaId { get; set; }
		[Required]
		public Guid CampusProgramId { get; set; }
		[Required]
		public Guid AdmissionTypeId { get; set; }
		[Required]
		public Guid GenderId { get; set; }
		[Required]
		public int MarkPercentage { get; set; }
		[Required]
		public DateTime MinPassingYear { get; set; }
		[Required]
		public DateTime FromDob { get; set; }
		[Required]
		public DateTime ToDob { get; set; }
		[Required]
		public int StatusId { get; set; }
		[Required]
		public String FullName{ get; set; }
		[Required]
		public String Gender{ get; set; }
		[Required]
		public String Description{ get; set; }
		[Required]
		public Guid LoggerId { get; set; }

		[Required]
		public Guid CampusId { get; set; }

		[Required]
		public Guid SessionId { get; set; }

    }
}